src/lib/GelatoOpsSDK.ts

```diff
@@ -201,7 +201,7 @@ export class GelatoOpsSDK {
       resolverData: args.resolverData,
       startTime: args.startTime,
       interval: args.interval,
-      proxy: args.proxy,
+      dedicatedMsgSender: args.dedicatedMsgSender,
```

Comment:
the `this._opsModule.encodeModuleArgs` function is still looking for arg `proxy`, which will not be found anymore